exchange/c2cx/.testConfig/.test.toml

```diff
@@ -0,0 +1,3 @@
+[c2cx]
+key = "EBE87C06-D0A2-4942-B2D3-57AD6E7C242A"
+secret = "4D7683C9-BE09-44B8-AE32-08070A2F7540"
```

Comment:
probably should not add this to the repo directly, we can add it to travis as a secret environment variable and add a travis step to write config file

also, the folder name should not be `.testConfig`, don't use mixed case for folder names

Reply:
I added these as envvars in the travis environment as C2CX_API_KEY and C2CX_API_SECRET, you can use them to write a config file during the travis setup

Reply:
You should squash your commits into one to remove these keys from the commit history